sp800-63a/sec4_ial.md

```diff
@@ -201,7 +201,7 @@ An enrollment code allows the CSP to confirm that the applicant controls an addr
 
 An enrollment code SHALL be comprised of one of the following:
 
-* Minimally, a random six character alphanumeric. For example, a code generated using a secure random cryptographic algorithm or a serial number for a physical hardware authenticator.
+* Minimally, a random six character alphanumeric. For example, a code generated using an approved random number generator or a serial number for a physical hardware authenticator.
```

Comment:
Maybe leave the word random off the first sentence, if we're not depending on the unguessability of the enrollment code.